### PR TITLE
feat: add to support to style id in Container

### PR DESCRIPTION
**Pull request by uziasferreirazup**
### Related Issues

#1419 

### Description and Example

Add support to style id in the component of layout.

### Checklist

Please, check if these important points are met using `[x]`:

- [X] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [X] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [X] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/f5f95216077cdd6765c12153d8deb7979649f280/android/beagle/src/main/java/br/com/zup/beagle/android/view/custom/BeagleFlexView.kt#L38">android/beagle/src/main/java/br/com/zup/beagle/android/view/custom/BeagleFlexView.kt#L38</a> - The constructor(rootView: RootView, style: Style, flexMapper: FlexMapper, viewRendererFactory: ViewRendererFactory, viewModel: ScreenContextViewModel, generateIdManager: GenerateIdManager, styleId: Int) has too many parameters. The current threshold is set to 7.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: android/beagle/src/main/java/br/com/zup/beagle/android/view/ViewFactory.kt [hunk 1]
```diff
@@ -49,6 +49,15 @@ internal class ViewFactory {
...
 
+    fun makeBeagleFlexView(
+        rootView: RootView,
+        style: Style,
+        styleId: Int,
+    ) = BeagleFlexView(
+        rootView, style,
+        styleId = styleId,
+    )
+
```
Comment (matheusribeirozup) [inline android/beagle/src/main/java/br/com/zup/beagle/android/view/ViewFactory.kt:60]:
```suggestion
    fun makeBeagleFlexView(
        rootView: RootView,
        style: Style,
        styleId: Int,
    ) = BeagleFlexView(
        rootView,
        style,
        styleId = styleId,
    )

```

File: backend/widgets/src/main/kotlin/br/com/zup/beagle/widget/layout/Container.kt [hunk 2]
```diff
@@ -29,9 +29,12 @@ import br.com.zup.beagle.widget.context.ContextData
...
  * when the Widget is displayed.
+ * @param styleId reference a native style in your local styles file to be applied on this container.
+ *
  */
 data class Container(
     val children: List<ServerDrivenComponent>,
     override val context: ContextData? = null,
-    val onInit: List<Action>? = null
+    val onInit: List<Action>? = null,
+    val styleId: String? = null
```
Comment (matheusribeirozup) [inline backend/widgets/src/main/kotlin/br/com/zup/beagle/widget/layout/Container.kt:39]:
```suggestion
    val styleId: String? = null,
```

File: android/beagle/src/main/java/br/com/zup/beagle/android/view/ViewFactory.kt [hunk 3]
```diff
@@ -49,6 +49,15 @@ internal class ViewFactory {
 
     fun makeBeagleFlexView(rootView: RootView, style: Style) = BeagleFlexView(rootView, style)
 
+    fun makeBeagleFlexView(
+        rootView: RootView,
+        style: Style,
+        styleId: Int,
+    ) = BeagleFlexView(
+        rootView, style,
+        styleId = styleId,
```
Comment (matheusribeirozup) [inline android/beagle/src/main/java/br/com/zup/beagle/android/view/ViewFactory.kt:58]:
```suggestion
        rootView,
        style,
        styleId = styleId,
```

**Review by paulomeurerzup — APPROVED** [review]

**Review by matheusribeirozup — APPROVED** [review]

**Review by paulomeurerzup — APPROVED** [review]

**Pull request merged**